### DISCUSSION TITLE
Support dot notation in labels display

### DISCUSSION THREAD

**Discussion by Fusseldieb** [directus/directus#4240] [Feature Requests]
When I setup a repeater in v9.0.0-rc19 like:
<img src="https://user-images.githubusercontent.com/4715129/100777282-5e256080-33e4-11eb-8fb6-9ef73896014f.png" width="250px">
Set up the display as "Labels" and then go over and fill out the repeater like:
<img src="https://user-images.githubusercontent.com/4715129/100777444-94fb7680-33e4-11eb-9b36-a2718a85b5b9.png" width="500px">
When I go over to view it, it shows nothing:
<img src="https://user-images.githubusercontent.com/4715129/100777541-b65c6280-33e4-11eb-8690-e008f438322f.png">

In v8, for instance, it showed a popup with the formatted values.
In v9, since it's mobile-friendly, I would suggest to put it onto the table without requiring a popup. (You can't really hover on mobile.)

**Comment by rijkvanzanten** [comment]
The current labels display "just" renders a tag for each value in an array of strings; it doesn't work for arrays with objects.

We can try extending the labels display to support dot notation, so you get to choose what to show for each row 🤔

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
